packages/ui/src/components/editor/DropdownItem/index.tsx

```diff
@@ -30,7 +30,7 @@ import { twMerge } from 'tailwind-merge'
 export interface EditorDropdownItemProps extends React.HTMLAttributes<HTMLDivElement> {
   selected?: boolean
   disabled?: boolean
-  ItemIcon?: ({ className }: { className: string }) => JSX.Element
+  ItemIcon?: (({ className }: { className?: string }) => JSX.Element) | React.ElementType
```

Comment:
the className is necessary because we want to apply `iconClassname` styles only to the icons